### DISCUSSION TITLE
Compose Feature

### DISCUSSION THREAD

**Discussion by trymeouteh** [podman-desktop/podman-desktop#5510] [Ideas]
- The ability to create compose files within the Podman Deskop GUI and save these files to your computer
- The ability to load compose files into Podman Desktop GUI
- The ability to Up and Down a compose file in Podman Desktop GUI
- The ability to choose podman compose or docker compose

Adding a button in the Images section called "Compose" which will take you to a compose menu were you can do all of the above in Podman Desktop

**Comment by aplatypus** [comment]
I like this.  Some constructive observations ...
 * After a compose file has been "`load`-ed"  ... That `pod` ought to be ably managed by the normal `Podman Desktop` run or pause, etc, commands.
